sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -280,10 +280,11 @@ def test_univariate_pls_regression():
 
     clf = pls_.PLSRegression()
     # Compare 1d to column vector
-    model1 = clf.fit(X, Y[:, 0]).coef_
-    model2 = clf.fit(X, Y[:, :1]).coef_
-    assert_array_almost_equal(model1, model2)
-
+    # issue #8856
+    ndim_predict_1D = clf.fit(X, Y[:, 0]).predict(X).ndim
+    ndim_predict_2D = clf.fit(X, Y[:, :1]).predict(X).ndim
+    assert_equal(ndim_predict_1D, 1)
```

Comment:
Try to follow PEP8 naming convention i.e. no capital letters in variable names: `ndim_predict_1d`